src/globalInfo/dxEnvVars.ts

```diff
@@ -0,0 +1,394 @@
+  'SFDX_AUTOUPDATE_DISABLE' = 'SFDX_AUTOUPDATE_DISABLE',
+  'SFDX_DISABLE_SOURCE_MEMBER_POLLING' = 'SFDX_DISABLE_SOURCE_MEMBER_POLLING',
+  'SFDX_DISABLE_TELEMETRY' = 'SFDX_DISABLE_TELEMETRY',
+  'SFDX_DNS_TIMEOUT' = 'SFDX_DNS_TIMEOUT',
+  'SFDX_DOMAIN_RETRY' = 'SFDX_DOMAIN_RETRY',
+  'SFDX_IMPROVED_CODE_COVERAGE' = 'SFDX_IMPROVED_CODE_COVERAGE',
+  'SFDX_INSTANCE_URL' = 'SFDX_INSTANCE_URL',
+  'SFDX_JSON_TO_STDOUT' = 'SFDX_JSON_TO_STDOUT',
+  'SFDX_LOG_LEVEL' = 'SFDX_LOG_LEVEL',
+  'SFDX_MAX_QUERY_LIMIT' = 'SFDX_MAX_QUERY_LIMIT',
+  'SFDX_MDAPI_TEMP_DIR' = 'SFDX_MDAPI_TEMP_DIR',
+  'SFDX_NPM_REGISTRY' = 'SFDX_NPM_REGISTRY',
+  'SFDX_PRECOMPILE_ENABLE' = 'SFDX_PRECOMPILE_ENABLE',
+  'SFDX_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE' = 'SFDX_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE',
+  'SFDX_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE' = 'SFDX_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE',
+  'SFDX_REST_DEPLOY' = 'SFDX_REST_DEPLOY',
+  'SFDX_SOURCE_MEMBER_POLLING_TIMEOUT' = 'SFDX_SOURCE_MEMBER_POLLING_TIMEOUT',
+  'SFDX_USE_GENERIC_UNIX_KEYCHAIN' = 'SFDX_USE_GENERIC_UNIX_KEYCHAIN',
+  'SFDX_USE_PROGRESS_BAR' = 'SFDX_USE_PROGRESS_BAR',
+  'SF_TARGET_ORG' = 'SF_TARGET_ORG',
+  'SF_TARGET_DEV_HUB' = 'SF_TARGET_DEV_HUB',
+  'SF_ACCESS_TOKEN' = 'SF_ACCESS_TOKEN',
+  'SF_API_VERSION' = 'SF_API_VERSION',
+  'SF_AUDIENCE_URL' = 'SF_AUDIENCE_URL',
+  'SF_CODE_COVERAGE_REQUIREMENT' = 'SF_CODE_COVERAGE_REQUIREMENT',
+  'SF_CONTENT_TYPE' = 'SF_CONTENT_TYPE',
+  'SF_DISABLE_AUTOUPDATE' = 'SF_DISABLE_AUTOUPDATE',
+  'SF_AUTOUPDATE_DISABLE' = 'SF_AUTOUPDATE_DISABLE',
+  'SF_DISABLE_SOURCE_MEMBER_POLLING' = 'SF_DISABLE_SOURCE_MEMBER_POLLING',
+  'SF_DISABLE_TELEMETRY' = 'SF_DISABLE_TELEMETRY',
+  'SF_DNS_TIMEOUT' = 'SF_DNS_TIMEOUT',
+  'SF_DOMAIN_RETRY' = 'SF_DOMAIN_RETRY',
+  'SF_IMPROVED_CODE_COVERAGE' = 'SF_IMPROVED_CODE_COVERAGE',
+  'SF_INSTANCE_URL' = 'SF_INSTANCE_URL',
+  'SF_JSON_TO_STDOUT' = 'SF_JSON_TO_STDOUT',
+  'SF_LOG_LEVEL' = 'SF_LOG_LEVEL',
+  'SF_MAX_QUERY_LIMIT' = 'SF_MAX_QUERY_LIMIT',
+  'SF_MDAPI_TEMP_DIR' = 'SF_MDAPI_TEMP_DIR',
+  'SF_NPM_REGISTRY' = 'SF_NPM_REGISTRY',
+  'SF_PRECOMPILE_ENABLE' = 'SF_PRECOMPILE_ENABLE',
+  'SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE' = 'SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE',
+  'SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE' = 'SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE',
+  'SF_REST_DEPLOY' = 'SF_REST_DEPLOY',
+  'SF_SOURCE_MEMBER_POLLING_TIMEOUT' = 'SF_SOURCE_MEMBER_POLLING_TIMEOUT',
+  'SF_USE_GENERIC_UNIX_KEYCHAIN' = 'SF_USE_GENERIC_UNIX_KEYCHAIN',
+  'SF_USE_PROGRESS_BAR' = 'SF_USE_PROGRESS_BAR',
+}
+type EnvMetaData = {
+  description: string;
+  synonymOf: Nullable<string>;
+};
+type EnvType = {
+  [key in EnvironmentVariable]: EnvMetaData;
+};
+
+const getMessage = (environmentVariable: EnvironmentVariable): string =>
+  messages.getMessage(camelCase(environmentVariable));
+
+export const DX_SUPPORTED_ENV_VARS: EnvType = {
```

Comment:
Why the `DX_` prefix?
Can we call this `SUPPORTED_ENV_VARS` and this file just `envVars.ts` instead of having `DX_` and `dx` as a prefix for the sake of simplicity or do you think it adds anything worthwhile?

Reply:
Removed dx prefix